cves/kernel/CVE-2014-4667.yml

```diff
@@ -116,359 +27,95 @@ upvotes_instructions: |
-    what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+  note: No stack traces were found
 forgotten_check:
-  question: |
-    Does the fix for the vulnerability involve adding a forgotten check?
-
-    A "forgotten check" can mean many things. It often manifests as the fix
-    inserting an entire if-statement or a conditional to an existing
-    if-statement. Or a call to a method that checks something.
-
-    Example of checks can include:
-      * null pointer checks
-      * check the current role, e.g. root
-      * boundary checks for a number
-      * consult file permissions
-      * check a return value
-
-    Answer must be true or false.
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+  answer: true
+  note: This bug forgot to check the emptiness of a list
 order_of_operations:
-  question: |
-    Does the fix for the vulnerability involve correcting an order of
-    operations?
-
-    This means the fix involves moving code around or changing the order of
-    how things are done.
-
-    Answer must be true or false.
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+  answer: false
+  note: No order of operations was changes
 lessons:
-  question: |
-    Are there any common lessons we have learned from class that apply to this
-    vulnerability? In other words, could this vulnerability serve as an example
-    of one of those lessons?
-
-    Leave "applies" blank or put false if you did not see that lesson (you do
-    not need to put a reason). Put "true" if you feel the lesson applies and put
-    a quick explanation of how it applies.
-
-    Don't feel the need to claim that ALL of these apply, but it's pretty likely
-    that one or two of them apply.
-
-    If you think of another lesson we covered in class that applies here, feel
-    free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: This could have been fixed by checking the input
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
     note:
   yagni:
-    applies:
+    applies: false
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: The inputs to this system are very complex
```

Comment:
Maybe elaborate a little bit more here and how it specifically relates to the vulnerability